src/db/index.js

```diff
@@ -2,6 +2,7 @@ const nodemailer = require('nodemailer');
 const generator = require('generate-password');
 const passwordHash = require('password-hash');
 const validator = require('email-validator');
+
```

Comment:
Linting?

Reply:
Just to trigger a coverage report to be generated. It can be reverted before we merge.

Reply:
@coopernetes - can revert?